opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/metrics/OpenCensusMetricsTest.java

```diff
@@ -26,7 +27,10 @@ class OpenCensusMetricsTest {
   void capturesOpenCensusAndOtelMetrics() throws InterruptedException {
     InMemoryMetricReader reader = InMemoryMetricReader.create();
     SdkMeterProvider otelMetrics =
-        SdkMeterProvider.builder().registerMetricReader(OpenCensusMetrics.attachTo(reader)).build();
+        SdkMeterProvider.builder()
+            .registerMetricReader(reader)
+            .registerMetricProducer(OpenCensusMetricProducer.create())
```

Comment:
This demonstrates how you would register a custom metric producer with SdkMeterProvider. Nice and tidy.

All the MetricData produced flow to any registered MetricReaders.

Reply:
cc @dashpole @jsuereth